backendRoot/src/main/java/com/virtuslab/gitmachete/backend/root/GitMacheteRepositoryBuilder.java

```diff
@@ -84,10 +84,8 @@ public IGitMacheteRepository build() throws GitMacheteException {
           .getOrElseThrow(e -> {
             Optional<Integer> errorLine = ((BranchLayoutException) e).getErrorLine();
             return new MacheteFileParseException(errorLine.isPresent()
-                ? String.format("Error occurred while parsing machete file %s in line %d",
-                    pathToBranchLayoutFile.toString(), errorLine.get())
-                : String.format("Error occurred while parsing machete file %s",
-                    pathToBranchLayoutFile.toString()),
+                ? "Error occurred while parsing machete file ${pathToBranchLayoutFile} in line ${errorLine.get()}"
```

Comment:
Can it be simplified now? Maybe the following:
`"Error occurred while parsing machete file ${pathToBranchLayoutFile}" + (errorLine.isPresent() ? "in line ${errorLine.get()}" : "")`